pkgs/development/libraries/curlcpp/default.nix

```diff
@@ -1,13 +1,14 @@
 { stdenv, fetchFromGitHub, cmake, curl }: 
 
-stdenv.mkDerivation {
-  name = "curlcpp-20170320";
+stdenv.mkDerivation rec {
+  name = "curlcpp-${version}";
+  version = "1.0";
 
   src = fetchFromGitHub {
     owner = "JosephP91";
     repo = "curlcpp";
-    rev = "f3c5499b312ba649b87f205e8c1f771958227a05";
-    sha256 = "0mwj5ghr04s6n3dkm50gp1aj0k30k3g4jkhhvq28vbqa0rmkyxk7";
+    rev = "bf0b97d48c4b17e5dc96ddfa0eabd4c2c9a74b21";
```

Comment:
You can refer to tags as well, which is preferable if there is one associated with the release.